File: csharp/odev04.cs
using System;
using System.Collections.Generic;

namespace _04
{
    class Program
    {
        static void Main(string[] args)
        {
            List<int> tek = new List<int>();
            List<int> cift = new List<int>();
            for (int i=0;  i<=10; i++)
            {
                if(i %2==0)
                {
                    cift.Add(i);
                }
                else
                {
                    tek.Add(i);
                }
            }
            foreach(int sayilar in tek)
            {
                Console.WriteLine(sayilar);
            }
            foreach (int sayilar in cift)
            {
                Console.WriteLine(sayilar);
            }

            Console.ReadLine();

        }
    }
}

File: python3/odev04.py
def cift_tek(sayi):
    if sayi %2 == 0:
        cift.append(sayi)
    else:
        tek.append(sayi)
    
    print("çift sayılar:")
    print(cift)
    print("tek sayılar:")
    print(tek)

sayilar=[]
cift=[]
tek=[]
for sayi in range(10):
    cift_tek(sayi)
    

    
